### ISSUE TITLE
Skip the "Tracks" section of the editor if there is only one track

### ISSUE THREAD

**Issue by oas777** [opencast/opencast-editor#1205]
Cf. https://oc.tobira.ethz.ch/editor-ui/index.html?mediaPackageId=25e82f02-db10-4ba6-937f-3252353cfbe8. 

![grafik](https://github.com/opencast/opencast-editor/assets/1752342/0fead42b-c30f-4f69-9553-d7cdec7cf3d2)

Or is there a reason to offer no option there?

**Comment by Arnei** [comment]
Nah, there is no reason to offer no option there. That was lazy coding, hoping that we get the #1009 sooner rather than later, because then it would make sense to show the track section for a single track provided it has both video and audio.

By "skip" I suppose you mean to not show the "Tracks" button in the left column at all if there is only one track, right?

**Comment by oas777** [comment]
Yes, don't show the "Tracks" button to the left.
